Test/QuickCheck/Arbitrary.hs

```diff
@@ -27,6 +27,7 @@ module Test.QuickCheck.Arbitrary
   , arbitraryBoundedEnum          -- :: (Bounded a, Enum a) => Gen a
   -- ** Helper functions for implementing shrink
 #ifndef NO_GENERICS
+  , genericArbitrary   -- :: (Generic a, GArbitrary (Rep a)) => Gen a
```

Comment:
Perhaps this should re-exported from `Test.QuickCheck`, since the functions below it are re-exported as well?